src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -459,74 +390,61 @@ private void DeployCode(ExecutionEnvironment env, TransactionSubstate substate,
             ExecutionEnvironment env,
             ulong intrinsicGas,
             ITxTracer txTracer)
         {
             if (_logger.IsEnabled(LogEventLevel.Verbose))
-            {
                 _logger.Verbose("Intrinsic gas calculated for {entry}: {intrinsicGas}", entry, intrinsicGas);
-            }
 
-            if (entry.GasLimit >= intrinsicGas)
-            {
-                return true;
-            }
-            
-            TraceLogInvalidTx(entry, $"GAS_LIMIT_BELOW_INTRINSIC_GAS {entry.GasLimit.ToString()} < {intrinsicGas.ToString()}");
+            if (entry.GasLimit >= intrinsicGas) return true;
+
+            TraceLogInvalidTx(entry,
+                $"GAS_LIMIT_BELOW_INTRINSIC_GAS {entry.GasLimit.ToString()} < {intrinsicGas.ToString()}");
             QuickFail(entry, env, txTracer);
             return false;
         }
 
-        private bool ValidateSender(ContractEntry entry, ExecutionEnvironment env, ITxTracer txTracer)
+        private bool ValidateSender(PublicEntry entry, ExecutionEnvironment env, ITxTracer txTracer)
         {
-            if (env.Sender != null)
-            {
-                return true;
-            }
-            
+            if (env.Sender != null) return true;
+
             TraceLogInvalidTx(entry, "SENDER_NOT_SPECIFIED");
             QuickFail(entry, env, txTracer);
             return false;
         }
 
-        private bool ValidateNonce(ContractEntry entry, ExecutionEnvironment env, ITxTracer txTracer)
+        private bool ValidateNonce(PublicEntry entry, ExecutionEnvironment env, ITxTracer txTracer)
         {
-            if (entry.Base.Nonce == _stateProvider.GetNonce(env.Sender))
-            {
-                return true;
-            }
-            
-            TraceLogInvalidTx(entry, $"WRONG_TRANSACTION_NONCE: {entry.Base.Nonce.ToString()} (expected {_stateProvider.GetNonce(env.Sender).ToString()})");
+            if (entry.Base.Nonce == _stateProvider.GetNonce(env.Sender)) return true;
+
+            TraceLogInvalidTx(entry,
+                $"WRONG_TRANSACTION_NONCE: {entry.Base.Nonce.ToString()} (expected {_stateProvider.GetNonce(env.Sender).ToString()})");
             QuickFail(entry, env, txTracer);
             return false;
         }
 
-        private bool ValidateSenderBalance(ContractEntry entry,
+        private bool ValidateSenderBalance(PublicEntry entry,
             ExecutionEnvironment env,
             ulong intrinsicGas,
             ITxTracer txTracer)
         {
             var senderBalance = _stateProvider.GetBalance(env.Sender);
-            if (intrinsicGas * env.GasPrice + env.Value <= senderBalance)
-            {
-                return true;
-            }
-            
-            TraceLogInvalidTx(entry, $"INSUFFICIENT_SENDER_BALANCE: ({env.Sender})_BALANCE = {senderBalance.ToString()}");
+            if (intrinsicGas * env.GasPrice + env.Value <= senderBalance) return true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)